src/models/schema.ts

```diff
@@ -49,6 +49,7 @@ export interface SchemaInterface extends BaseModel<v2.AsyncAPISchemaObject>, Ext
   propertyNames(): SchemaInterface | undefined;
   readOnly(): boolean | undefined;
   required(): Array<string> | undefined;
+  schemaFormat(): string
```

Comment:
Didnt think about it until now, but it's so similar to `$schema` just with different values 🙃

Reply:
$schema is metadata of `JSON Schema`. These two fields should not be "merged".

Reply:
Agree 😄 

But it's just who very similar values in some cases and in others not at all, weird combo to have.

Reply:
Feels like a hack, but no way around it when we transform everything to JSON Schema 😆

Reply:
Not a big issue imho. We could create a model that inherits from Schema but includes that method, but at the end is the same.